src/intl/fr/page-staking.json

```diff
@@ -196,5 +215,6 @@
   "page-staking-meta-title": "Mise en jeu sur Ethereum",
   "page-staking-withdrawals-important-notices": "Informations importantes",
   "page-staking-withdrawals-important-notices-desc": "Les retraits ne sont pas encore disponibles. Veuillez lire la <a href=\"https://blog.ethereum.org/2021/12/01/eth2-merge-and-post-merge-faq/\" target=\"_blank\">FAQ de Merge Eth2 et post-merge</a> pour plus d'informations.",
```

Comment:
we were removing this url https://blog.ethereum.org/2021/12/01/eth2-merge-and-post-merge-faq/ later right? 404

Reply:
Yeah, that would fall under https://github.com/ethereum/ethereum-org-website/issues/10003 I believe